simple/util/dc_client.py

```diff
@@ -217,3 +252,28 @@ def post(path: str, data={}) -> dict:
         f'{resp.status_code}: {resp.reason}\n{response["message"]}\nRequest: {path}\n{data}'
     )
   return response
+
+
+async def post_async(client: AsyncClient, path: str, data={}) -> dict:
```

Comment:
@pradh maybe the website nl code can use this asyncClient to do concurrent api call.